e2e/cypress/integration/team/teams_spec.js

```diff
@@ -75,26 +74,26 @@ describe('Teams Suite', () => {
         // # Create team
         cy.createNewTeam(teamName, teamURL);
 
-        // # Click hamburger menu > “Add Members to Team”
+        // # Click hamburger menu > Invite People
         cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
-        cy.get('#sidebarDropdownMenu #addUsersToTeam').should('be.visible').click();
+        cy.get('#invitePeople').should('be.visible').and('have.text', 'Invite People').click();
 
-        // * Check that the "add new members modal" opened up
-        cy.get('#addUsersToTeamModal').should('be.visible');
+        // * Check that the Invitation Modal opened up
+        cy.getByTestId('invitationModal', {timeout: TIMEOUTS.TINY}).should('be.visible');
 
-        // # In "Add New Members To [team name] Team" modal, type the first few letters of a user who is on that server but not on the active team in the search box to filter the list
-        cy.focused().type(nameStartsWith, {force: true});
+        cy.getByTestId('inputPlaceholder').should('be.visible').within(($el) => {
+            // # Type the first letters of a user
+            cy.wrap($el).get('input').type(nameStartsWith, {force: true});
 
-        // * Verify list filters as expected
-        cy.get('.filtered-user-list').should('contain', user.username);
+            // * Verify user is on the list, then select by clicking on it
+            cy.wrap($el).get('.users-emails-input__menu').
+                children().should('have.length', 1).
+                eq(0).should('have.text', `@${user.username} - ${user.firstName} ${user.lastName}`).
+                click();
+        });
 
-        // # Select a user from the list, verify number of users who can be added decrements from 20 to 19
-        cy.get('#numPeopleRemaining').should('contain', maxSelectableValues);
-        cy.focused().type('{enter}');
-        cy.get('#numPeopleRemaining').should('contain', maxSelectableValues - 1);
-
-        // # Click Add
-        cy.get('#saveItems').click();
+        // # Click Invite Members
+        cy.getByText(/Invite Members/).click();
```

Comment:
I like this approach as it feels like closer to a "user looking at the text they wanting to click".  Also, this one will not need to update Jest snapshots whenever where adding test ID.